src/System.Security.Cryptography.X509Certificates/ref/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "System.Collections.NonGeneric": "4.4.0-beta-24717-02",
```

Comment:
Correct S.Collections.NonGenerics is part of netstandard2.0 so it will be in the shared framework.